app/views/physical_rack/show.html.haml

```diff
@@ -2,12 +2,20 @@
   - if %w(physical_racks physical_servers).include?(@display)
     = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
     - if %w(physical_servers).include?(@display)
-      %physical-server-toolbar#ems_physical_infra_show_list_form
+      %physical-server-toolbar#physical_rack_show_form
       :javascript
-        miq_bootstrap('#ems_physical_infra_show_list_form')
+        miq_bootstrap('#physical_rack_show_form')
   - else
     - case @showtype
     - when "main"
       = render :partial => 'layouts/textual_groups_generic'
     - when "timeline"
       = render :partial => "layouts/tl_show_async"
+
+
+= render :partial => "layouts/toolbar_actions"
+
+%physical-rack-toolbar#physical_rack_show_form{'physical-rack-id' => @record.id}
```

Comment:
This looks like dead code... any reason to it? (As far as I know, you're not adding a physicalRackToolbar component in this PR, nor does one exist.)

Reply:
You are right! :)